src/db.c

```diff
@@ -1523,7 +1525,7 @@ db_build_query_items(struct query_params *qp)
     return NULL;
 
   count = sqlite3_mprintf("SELECT COUNT(*) FROM files f %s;", qc->where);
-  query = sqlite3_mprintf("SELECT f.* FROM files f %s %s %s;", qc->where, qc->order, qc->index);
+  query = sqlite3_mprintf("SELECT f.* FROM files f %s %s %s %s;", qc->where, qc->group, qc->order, qc->index);
```

Comment:
What does sqlite3_mprintf do if qc->group is null? Print "(null)" in the string? I see qc->order and qc->index get set to empty strings by db_build_query_clause().

Reply:
I changed it, now qc->group is set to empty string in case no group clause is needed.